packages/core/src/repositories/documentLogsWithMetadataAndErrorsRepository/index.ts

```diff
@@ -24,13 +25,15 @@ export class DocumentLogsWithMetadataAndErrorsRepository extends Repository<Docu
       .select({
         ...getTableColumns(documentLogs),
         commit: getTableColumns(commits),
+        // TODO: Denormalize these aggregations and persist them at write time
         tokens: sum(providerLogs.tokens).mapWith(Number).as('tokens'),
         duration: sum(providerLogs.duration)
           .mapWith(Number)
           .as('duration_in_ms'),
         costInMillicents: sum(providerLogs.costInMillicents)
           .mapWith(Number)
           .as('cost_in_millicents'),
+        // TODO: Denormalize the errors and persist them at write time
```

Comment:
The plan is to get rid of the errors table and put there in the logs and eval results?

Reply:
We will remove the table yes but we will also migrate for doc/provider logs to traces and spans